libraries/chain/include/eosio/chain/database_utils.hpp

```diff
@@ -183,7 +171,8 @@ namespace fc {
    inline
    void from_variant( const variant& v, eosio::chain::shared_blob& b ) {
       std::string _s = base64_decode(v.as_string());
-      b = eosio::chain::shared_blob(_s.begin(), _s.end(), b.get_allocator());
+      std::destroy_at(std::addressof(b));
+      new (std::addressof(b)) eosio::chain::shared_blob(_s.begin(), _s.end());
```

Comment:
No, because the `shared_blob` class doesn't have a constructor taking std::string (even though its base class does). I thought of adding one, but I thought this was the least intrusive change.

Reply:
I think we should add the constructor as this is rather ugly to do here.

Reply:
But I just added a `std::string_view` constructor to `shared_blob` so it works now.

Reply:
I did!